### PR TITLE
corerad: 0.2.7 -> 0.2.8

### PR DESCRIPTION
**Pull request by mdlayher**
###### Motivation for this change

New CoreRAD release, special handling of SIGHUP.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Builds and runs, tests continue to pass:

```
$ ./result/bin/corerad -h
CoreRAD v0.2.8 BETA (2020-08-08)
flags:
  -c string
        path to configuration file (default "corerad.toml")
  -init
        write out a default configuration file to "corerad.toml" and exit
```

/cc @jonringer @marsam @zowoq (thanks for help on previous PRs!)

### CODE REVIEW

**Comment by mdlayher** [comment]
@GrahamcOfBorg test corerad

File: nixos/modules/services/networking/corerad.nix [hunk 1]
```diff
@@ -81,6 +81,7 @@ in {
         NotifyAccess = "main";
         ExecStart = "${getBin cfg.package}/bin/corerad -c=${cfg.configFile}";
         Restart = "on-failure";
+        RestartKillSignal = "SIGHUP";
```
Comment (jonringer) [inline nixos/modules/services/networking/corerad.nix:84]: this should probably be a different commit:

```
corerad: 0.2.7 -> 0.2.8
nixos/corerad: set RestartKillSignal
```
not sure what the subject for the module should be.

Comment (mdlayher) [reply nixos/modules/services/networking/corerad.nix:84]: Not a problem, done!

**Comment by mdlayher** [comment]
@GrahamcOfBorg test corerad

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 94953 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  - corerad</details>
